### ISSUE TITLE
servant-elm-0.4.0.0 test suite failure with latest servant

### ISSUE THREAD

**Issue by DanBurton** [mattjbray/servant-elm#26]
As seen on the stackage nightly build. This is presumably related to us upgrading to the latest version of `servant`.

```
[2 of 2] Compiling Main             ( test/GenerateSpec.hs, dist/build/servant-elm-test/servant-elm-test-tmp/Main.o )
Linking dist/build/servant-elm-test/servant-elm-test ...
> /tmp/stackage-build13/servant-elm-0.4.0.0$ dist/build/servant-elm-test/servant-elm-test

encoding a simple api
  does it FAILED [1]
  with dynamic URLs

Failures:

  test/GenerateSpec.hs:98:
  1) encoding a simple api does it
       < generated
       > test/elm-sources/getWithaheaderSource.elm
       7,8c7,8
       < getWith-a-header : String -> Int -> Http.Request (String)
       < getWith-a-header header_myStringHeader header_MyIntHeader =
       ---
       > getWithaheader : String -> Int -> Http.Request (String)
       > getWithaheader header_myStringHeader header_MyIntHeader =

Randomized with seed 977727191

Finished in 0.0083 seconds
2 examples, 1 failure
```

**Comment by DanBurton** [comment]
We've temporarily disabled `servant-elm` from the stackage nightly builds. We'll wait until you've uploaded a new version of `servant-elm` before re-enabling it. Alternatively, at your request we can re-enable `servant-elm` immediately and mark this as an expected test failure.

**Comment by mattjbray** [comment]
@DanBurton Thanks, fixed in `0.4.0.1`.

**Comment by DanBurton** [comment]
Sounds good, I've re-enabled it on the stackage nightly build plan.
